tavla/app/(admin)/edit/[id]/components/Footer/actions.ts

```diff
@@ -18,18 +19,21 @@ export async function setFooter(bid: TBoardID, data: FormData) {
 
     const footerText = data.get('footer') as string
```

Comment:
Generelt -> kan vi endre denne variabelen (både i formet og her) til infoMessage eller message eller bare text? Og ikke footer?

Reply:
jeg er enig i å endre bort fra database-navnene, tenker det er verdt det for å gjøre koden mer lesbar